src/Mentat.UI/Views/Home/Index.cshtml

```diff
@@ -1,12 +1,59 @@
-﻿@{
+﻿@model Mentat.UI.ViewModels.CardViewModel
+
+@{
     ViewData["Title"] = "Home";
 }
 
-<div class="container d-flex vh-60">
+<div class="container">
     <span class="dot"></span>
+    
+    <div class="container pt-5">
+        <div id="myCarousel">
+            <div id="carouselCard" class="carousel slide" data-ride="carousel">
+
+              <!-- Card Indicator -->
+              <ul class="carousel-indicators">
+                @{int j = 0;}
+                @foreach (var item in Model.cards)
+                {
+                    var jActive = j == 0 ? "active" : "";
+                    <li data-target="#carouselCard" data-slide-to="@j" class="@jActive"></li>
+                    j++;
+                }
+              </ul>
 
-    <div class="wrapper my-auto">
-        <div class="row d-flex">
+              <!-- Cards -->
+              <div class="carousel-inner">
+                  @{int i = 0;}
+                  @foreach (var item in Model.cards)
+                 {
+                    i++;
```

Comment:
This should go at the end of the loop like the other one. Be consistent. Better yet do a for-loop